mappings/net/minecraft/util/shape/AbstractVoxelShapeContainer.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS cmm net/minecraft/util/shape/AbstractVoxelShapeContainer
+	CLASS cmm$b
+	FIELD a xSize I
+	FIELD b ySize I
+	FIELD c zSize I
+	FIELD d AXES [Ley$a;
+	METHOD <init> (III)V
+		ARG 1 xSize
+		ARG 2 ySize
+		ARG 3 zSize
+	METHOD a isEmpty ()Z
+	METHOD a modify (IIIZZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 resize
+		ARG 5 included
+	METHOD a getAxisMin (Ley$a;)I
```

Comment:
getMinAxis/getXSize? getAxisMin/getSizeX? :thinking:

Reply:
I'm not sure about this. It should definitely be getXSize, as that's how you'd order the words in english, but getMinAxis sounds wrong. What the method does is get the minimum/maximum coordinate in the shape on the given axis. getAxisMin makes sense because, if you didn't have a parameter and instead had individual methods, they'd be getXMin, getXMax, etc.

Reply:
That works. Updated.